cmd/influx_tools/export/README.md

```diff
@@ -73,4 +73,99 @@ Seq #           ID              Start                           End
 
 A range can either be a single sequence number or an interval as shown previously.
 
-**Hint**: Include the `-print-only` option to display the plan and exit without exporting any data. 
\ No newline at end of file
+**Hint**: Include the `-print-only` option to display the plan and exit without exporting any data.
+
+Example usage for Enterprise
+----------------------------
+
+Note: This is only for fully replicated clusters. For non-fully-replicated clusters (RF < # of nodes),
+the current best method would be exporting in line protocol and re-importing after deleting the old shards.
+
+This example takes many small shards and collapses them to fewer large shards, which can be beneficial
+for TSI index performance.
+
+1) Create collapsedir containing export/import data from all shards:
+
+This can be done on a running backup of the cluster - `influx_tools` needs to be able
```

Comment:
Should this be reworded into a recommendation instead of an option? i.e. "it is recommended that this be done on a running backup..."

Also, the bit about influx_tools needing to query metadata and disk state could probably be removed for clarity.

Also, these two sentences would probably be clearer if they were bullets instead of a paragraph

Reply:
I think it's important to know that the tool needs both running meta nodes and access to the disk of a data node - that's a weird combination. Basically the tool runs 'as-if' it were the data node. I'll try to re-word.